### PR TITLE
Fix some typo in 2015-11

### PR DESCRIPTION
**Pull request by azu**
Fixes some typo and fix indent.

/cc @rwaldron @ljharb

### CODE REVIEW

File: es7/2015-11/nov-19.md [hunk 1]
```diff
@@ -673,7 +673,7 @@ JHD: the risk grows the longer we wait.
 DD: it sounds like more information is needed.
 
 #### Conclusion/Resolution
-    - More research
+- More research
```
Comment (azu) [inline es7/2015-11/nov-19.md]: tc39-notes 2015-12-29 22-01-14

`<pre>`  => `<li>`

**Comment by ljharb** [comment]
LGTM

**Comment by azu** [comment]
ping @rwaldron

**Comment by azu** [comment]
ping @rwaldron ?

**Comment by rwaldron** [comment]
Thanks for the fix up—always appreciated :)

**Pull request merged**